src/ui/Profile/Names.svelte

```diff
@@ -7,7 +7,7 @@
   export let followers: number | undefined = undefined
 
   $: ({ username, email } = user)
-  $: subtitle = followers >= 0 ? `${followers} Followers` : email
+  $: subtitle = followers && followers >= 0 ? `${followers} Followers` : email
```

Comment:
`&& followers` is redundant
```suggestion
  $: subtitle = followers >= 0 ? `${followers} Followers` : email
```